controllers/databasecluster_controller.go

```diff
@@ -1566,8 +1566,13 @@ func reconcilePGBackRestRepos(
 			// Keep track of backup storages which are already in use by a repo
 			backupStoragesInRepos[backupSchedule.BackupStorageName] = struct{}{}
 
+			copies := int(backupSchedule.RetentionCopies)
+			if copies == 0 {
+				// Zero copies means unlimited. PGBackRest supports values 1-9999999
+				copies = 9999999
+			}
```

Comment:
to avoid using the same magic constant twice we may have it in a separate function

Reply:
We could, but given we ignore code reuse all over operator, I just decided to follow the same pattern :)
Fixed now